src/utils/util.js

```diff
@@ -97,6 +107,18 @@ function isPluginUrl(serverUrl, inputURL) {
       parsedURL.pathname.toLowerCase().startsWith('/plugins/')));
 }
 
+function isTrustedRemoteUrl(serverUrl, inputURL) {
+  const server = getServerInfo(serverUrl);
+  const parsedURL = parseURL(inputURL);
+  if (!parsedURL || !server) {
+    return false;
+  }
+  return (
+    equalUrlsIgnoringSubpath(server, parsedURL) &&
+    (parsedURL.pathname.toLowerCase().startsWith(`${server.subpath}trusted/`) ||
+      parsedURL.pathname.toLowerCase().startsWith('/trusted/')));
```

Comment:
in our previous conversations I didn't raise this as I thought it was temporary to make sure it went through, but we can't trust a url just because it says `trusted` in it.

one possible way is to add `trustedOrigins` to the config setup by https://github.com/mattermost/desktop/blob/master/src/common/config/buildConfig.js This way, it would have to be compiled in. if we go this way, I'd create a followup ticket to also have this option within GPO directives for windows.

Reply:
Ah, origin is same as server. This is url suffix like https://community.mattermost.com/trusted/
Same as (already implemented in code) https://community.mattermost.com/plugins/

Reply:
The origin check is ok, but I don't understand how the server-side part of this is meant to work. How is the `/trusted/` path created on the server and what does the page contain?

Reply:
Well in my case I created /trusted/ on server side (nginx redirection etc.). In case of somebody else, you can create your own content on /trusted/ path. If you want to take deep look, I can provide you with server and username, so you can look for yourself. To make long story short, I modied Jitsi Mattermost plugin in Go, in order to support this. (Just simple creating of URL with JWT).